src/App.js

```diff
@@ -1,18 +1,35 @@
 import './App.css';
-import initialCards from './cards-data';
+// import initialCards from './cards-data';
 import Player from './components/Player';
 import CardList from './components/CardList';
-import { useState } from 'react';
+// import { useContext, useState } from 'react';
 import ExecutePassButton from './components/ExecutePassButton';
+import { useGameContext } from './context/GameContext.js';
 
 function App() {
-  const [deck, setDeck] = useState(initialCards);
-  const [playerOneHand, setPlayerOneHand] = useState([]);
-  const [selectedCard, setSelectedCard] = useState();
-  const [playerTwoHand, setPlayerTwoHand] = useState([]);
-  const [playerThreeHand, setPlayerThreeHand] = useState([]);
-  const [from, setFrom] = useState('deck');
-  const [to, setTo] = useState(1);
+  const {
+    deck,
+    setDeck,
+    playerOneHand,
+    setPlayerOneHand,
+    playerTwoHand,
+    setPlayerTwoHand,
+    playerThreeHand,
+    setPlayerThreeHand,
+    selectedCard,
+    setSelectedCard,
+    from,
+    setFrom,
+    to,
```

Comment:
is ```App.js``` using all of these? I might be wrong. If not you'll only want to be importing those that are needed in the file that's doing the importing